datasets/gov/migracija/lt_pilietybe.csv

```diff
@@ -0,0 +1,31 @@
+id,dataset,resource,base,model,property,type,ref,source,prepare,level,access,uri,title,description
+,datasets/gov/migracija/lt_pilietybe,,,,,,,,,,,,,Duomenys apie asmenis
+,,,,,,prefix,spinta,,,,,https://github.com/atviriduomenys/spinta/issues/,,
+,,,,,,,,,,,,,,
+,,,,Prasymas,,,id,,,4,open,,Prašymai dėl Lietuvos Respublikos pilietybės,
+,,,,,id,string,,,,4,open,,Unikalus identifikatorius,
+,,,,,pateikimas,date,D,,,4,open,,Prašymo pateikimo data,
+,,,,,prasymo_registracija,date,D,,,4,open,,Prašymo registracijos data,
+,,,,,pagrindas,string,,,,3,open,,Prašymo pagrindas,
+,,,,,pilietybe,string,,,,2,open,,Asmens pilietybė,
+,,,,,,comment,property,,"""update(property: """"pilietybe[]""""",,,,,
+,,,,,amziaus_grupe,string,,,,3,open,,Asmens amžiaus grupė pagal gimimo datą,
+,,,,,lytis,string,,,,4,,,Lytis byloje,
+,,,,,,enum,,,"""M""",,open,,Moteris,
+,,,,,,,,,"""V""",,,,Vyras,
+,,,,,skubus_prasymas,boolean,,,,4,open,,Prašymo nagrinėjimo tvarka,
+,,,,,skyrius,string,,,,4,open,,Prašymo būsenos sukūrimo vieta,
+,,,,,,,,,,,,,,
+,,,,Sprendimas,,,id,,,4,open,,Sprendimai dėl Lietuvos Respublikos pilietybės suteikimo,
+,,,,,id,string,,,,4,open,,Unikalus identifikatorius,
+,,,,,sprendimo_registracija,date,D,,,4,open,,Sprendimo registracijos data,
+,,,,,sprendimas,string,,,,3,open,,Migracijos departamento sprendimas dėl Lietuvos Respublikos pilietybės,
+,,,,,sprendimo_pagrindas,string,,,,3,open,,Sprendimo pagrindas,
+,,,,,poreiksme,string,,,,3,open,,Sprendimo poreikšmė,
+,,,,,,comment,property,,"""update(property: """"poreiksme[]""""",,,,,
```

Comment:
Pirminiu vertinimu, tai ne `type=array` duomenys.
Ką reiškia šis komentaras?

jeigu vis gi tai tai `type=array`, tai `property=poreiksme` brandos lygis turėtų būti 2, nes : "Nestandartiniai duomenų tipai - antru brandos lygiu žymimi duomenys, kurių nurodytas tipas neatitinka realaus duomenų tipo"

Reply:
![image](https://user-images.githubusercontent.com/73559010/217556300-4bd15726-1ed3-4988-a7ee-e9ce15d3d505.png)
Jūs teisi, brandos lygis turėtų būti 2 šiuo atveju. Vis dėlto stulpelis yra array, kadangi į jį buvo sutrauktos kelios poreikšmės tam pačiam asmeniui. Sutinku, tokių atvejų yra nedaug. Įkėliau vieną pavyzdį prisegtame paveiksliuke ("asmenys Lietuvos Respublikos pilietybę gali atkurti, jeigu jie nėra kitos valstybės piliečiai. Reikalavimas atsisakyti kitos valstybės pilietybės netaikomas asmenims, kurie pagal PĮ 7 straipsnio 2, 3 ar 4 punktą gali būti kartu Lietuvos Respublikos ir kitos valstybės piliečiai; asmuo, iki 1940 m. birželio 15 d. turėjęs Lietuvos Respublikos pilietybę, ir jo palikuonys, kurie iki šio įstatymo įsigaliojimo nėra įgiję Lietuvos Respublikos pilietybės, turi neterminuotą teisę atkurti Lietuvos Respublikos pilietybę, neatsižvelgiant į tai, kokioje valstybėje  Lietuvos Respublikoje ar kitoje valstybėje  jie nuolat gyvena"). Čia skirtingos reikšmės atskirtos kabliataškiu (;). Sutinku, kad tokiame pavyzdyje sunku įžvelgti kelias sujungtas reikšmes. Gal man reikėtų verčiau ateityje array tipų stulpeliams naudoti kitokius jungtukus (vietoj kabliataškio kokį kitą ženklą)?